spec/inputs/base/validations_spec.rb

```diff
@@ -338,5 +338,145 @@ class MyInput
       end
     end
   end
+
+  describe '#validation_min' do
+    before :example do
+      allow(instance).to receive(:validations?).and_return(:true)
+      allow(instance).to receive(:validations).and_return([validator])
```

Comment:
This is stubbing the object under test. It should create a model that has validations and use that instead.